src/solc_0.8/raffle/PeopleOfCrypto.sol

```diff
@@ -207,6 +207,10 @@ contract PeopleOfCrypto is ERC721Enumerable, Ownable, ReentrancyGuard {
         return value;
     }
 
+    function getPersonalizationTraits(uint256 _tokenId) external view returns (uint256) {
```

Comment:
@ioanaverebi I think we should keep the naming convention used in ERC721
I would name the method
`personalizationOf`